### ISSUE TITLE
ElectrumX server connection issues

### ISSUE THREAD

**Issue by smk762** [namecoin/electrum-nmc#336]
As part of our work towards ensuring coins listed in our [AtomicDEX app](https://atomicdex.io/en/) are functional, I've implemented a dashboard & API to track the status of electrum servers listed in https://github.com/KomodoPlatform/coins/tree/master/electrums

Dashboard: https://stats.kmd.io/atomicdex/electrum_status/
API: https://electrum-status.dragonhound.info/api/v1/electrums_status

We detected 5/6 of the NMC servers we have listed are failing. Please let me know once it has been resolved, or if some servers are to be removed.

If you would like some help with setting up telegram / discord alerts to warn when servers are failing, we are happy to assist.

**Comment by gits7r** [comment]
You are right. I have done PR #365 to address this issue. Hopefully we can get it merged. I have opened the PR against master branch, but I saw 4.0.6 branch is more recently updated, so the maintainers should properly merge and cherry pick if needed.

**Comment by gits7r** [comment]
@smk762 can you patch KomodoPlatform/coins/blob/master/electrums/NMC so that it reflects the servers in PR #365 please? Those servers are very old, you can run an electrum-nmc client instance to see the servers advertised via the peer to peer mechanism, otherwise the one in PR #365 should suffice.

**Comment by smk762** [comment]
Can you indicate which (if any) run on SSL/WSS?

**Comment by gits7r** [comment]
Yes, all of them run TCP in port 50001, SSL in port 50002 , WS on 50003 and WSS on 50004 -- the SSL certificate is self signed.
When I say all of them I mean all of the ones added via PR #365 , that I know for sure, but others could support SSL/WSS as well however did no check, for the ones in in the PR I have checked personally.
